vuestate/src/store/modules/store.js

```diff
@@ -1,29 +1,69 @@
+import Vue from "vue";
 export default {
     state: {
         todos: [],
-
+        //sorting key & order
+        sort_by: 'sno',
+        sort_desc: true,
     },
     mutations: {
-        addtodo: (state,item) =>{
-            // console.log(item)
-            state.todos.push(item)
-            console.log(state.todos)
+        addtodo: (state, item) => {
+            // console.log(item.key01)
+            state.todos.push({
```

Comment:
this should be like state.todos.push(item)
this simple. pass proper json from component